src/coreclr/vm/ilstubcache.cpp

```diff
@@ -562,7 +526,7 @@ MethodDesc* ILStubCache::GetStubMethodDesc(
     RETURN pMD;
 }
 
-void ILStubCache::DeleteEntry(void* pParams)
+void ILStubCache::DeleteEntry(ILStubHashBlob* pHashBlob)
```

Comment:
Is this used? Where is it called from?

Reply:
It is used by `NDirect::RemoveILStubCacheEntry` (when there is an error during stub generation):
https://github.com/dotnet/runtime/blob/2f2593177dafbe702407fe0b7ac156a7829b7ee6/src/coreclr/vm/dllimport.cpp#L3992-L3994